packages/cli/README-source.md

```diff
@@ -1,20 +1,21 @@
 <h1 align="center">
-  <a href="https://zapier.com"><img src="https://cdn.rawgit.com/zapier/zapier-platform-cli/master/goodies/zapier-logomark.png" alt="Zapier" width="200"></a>
+  <a href="https://zapier.com"><img src="https://raw.githubusercontent.com/zapier/zapier-platform/master/packages/cli/goodies/zapier-logomark.png" alt="Zapier" width="200"></a>
   <br>
   Zapier Platform CLI
   <br>
   <br>
 </h1>
 
 <p align="center">
-  <a href="https://travis-ci.org/zapier/zapier-platform-cli"><img src="https://img.shields.io/travis/zapier/zapier-platform-cli/master.svg" alt="Travis"></a>
+  <!-- this isn't live yet? or won't be until I push -->
```

Comment:
it looks like this works now that i've pushed. if that's all ✅, i'll uncomment and update accordingly. 

https://travis-ci.org/zapier/zapier-platform/jobs/589269085